google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/debugger/CloudDebugSettingsEditor.java

```diff
@@ -47,7 +60,21 @@ protected JComponent createEditor() {
   }
 
   @Override
-  protected void resetEditorFrom(CloudDebugRunConfiguration runConfiguration) {
-    settingsPanel.setProjectName(runConfiguration.getCloudProjectName());
+  protected void resetEditorFrom(@NotNull CloudDebugRunConfiguration runConfiguration) {
+    String projectId = runConfiguration.getCloudProjectName();
+    if (Strings.isNullOrEmpty(projectId)) {
+      settingsPanel.getProjectSelector().setSelectedProject(null);
+    } else {
+      settingsPanel
+          .getProjectSelector()
+          .setSelectedProject(
+              CloudProject.create(
+                  projectId,
+                  projectId,
+                  null,
+                  Optional.ofNullable(Services.getLoginService().getActiveUser())
+                      .map(CredentialedUser::getEmail)
+                      .orElse("")));
```

Comment:
is it valid to call `create` with an empty string for the user name?

Reply:
It is a valid case for `ProjectSelector` - when user name is empty, it will show `Select Google Cloud project` prompt assuming project couldn't be found or bound to logged in user.